thunder/transforms/quantization.py

```diff
@@ -294,3 +295,214 @@ def transform_traces_pre_prologue(self, prologue_trace, computation_trace, epilo
 
         new_computation_trace.set_provenance(thunder.core.trace.TraceProvenance("quant pass"))
         return prologue_trace, new_computation_trace, epilogue_trace
+
+
+class LORATransform(Transform):
```

Comment:
Should we put LORATransform in its own file?

Reply:
Let me get to this after resolving the other comments :)